packages/core/contracts/polygon/OracleChildTunnel.sol

```diff
@@ -41,7 +42,7 @@ contract OracleChildTunnel is OracleBaseTunnel, OracleAncillaryInterface, FxBase
         bytes32 identifier,
         uint256 time,
         bytes memory ancillaryData
-    ) public override onlyRegisteredContract() {
+    ) public override nonReentrant() onlyRegisteredContract() {
```

Comment:
- The contracts do not use reentrancy guards. We have not identified a scenario that requires reentrancy protection, but we thought it is worth mentioning because reentrancy guards are used extensively throughout the code base, so this may be an oversight.